md/techncial-requirements-day-11.md

```diff
@@ -1,23 +1,36 @@
 # Technical Requirements for Busmall:
 
+* ### An HTML page that has divs for:
 * Display welcome and instructions at top of page
-
 * Display a set of 3 images
+* Display results in a list
 
-* Each set of images should contain no duplicates
+* ### Have a product object constructor that contains:
+* Product name
+* image
+* Times clicked
+* Times shown
 
-* Each set should have 3 different images from the previous set of images
+* These product objects will be stored in an object that will use the product name as the key
 
-* For each image displayed, update the number of times it has been shown
+* Make a function to choose a random image
 
-* For each image clicked, update the number of times it has been clicked.
+* Each set of images should contain no duplicates so make an array to hold current images to ensure that each image chosen at random doesn't duplicate within its group
+
+* Each set should have 3 different images from the previous set of images so create an array to hold the previous 3 images for comparison to current images
+
+* Function to handle comparison of randomly generated image to images in current and previous arrays
```

Comment:
Since this is a significant logic piece, add another sentence to this one saying how you'll do the comparison